eng/scripts/job-matrix/README.md

```diff
@@ -38,20 +38,35 @@ jobs:
       - Name: base_product_matrix
         Path: /eng/pipelines/matrix.json
         Selection: sparse
+		RunOnVM: true
```

Comment:
These don't match the names in the actual implementation.

Reply:
Ah, older name, forgot to change here.